### PR TITLE
Library list - TabBar creation

### PR DESCRIPTION
**Pull request by emmanuelhcpk**
Sorry for the name of the branch

### CODE REVIEW

File: Project/ViewController.swift [hunk 1]
```diff
@@ -8,19 +8,39 @@
...
+        wishlistTab.setup()
+        // Create AddBook
+        let addBookTab = AddBookViewController()
+        addBookTab.setup()
+        // Create Rentals
+        let rentalsTab = RentalsViewController()
+        rentalsTab.setup()
+        // Create Settings
+        let settingsTab = SettingsViewController()
+        settingsTab.setup()
```
Comment (acid-wolox) [inline Project/ViewController.swift]: I noticed you created a `setup` method for every Controller. I think you should just cut the code in the setup method and paste it in the `viewDidLoad` or `init` methods. Remember: in OOP, the class should set everything up for their new instances (whenever possible, that is).

File: Project/ViewController.swift [hunk 2]
```diff
@@ -8,19 +8,39 @@
...
+        self.viewControllers = [libraryTab, wishlistTab,addBookTab,rentalsTab,settingsTab]
```
Comment (acid-wolox) [inline Project/ViewController.swift]: Maybe the whole section can be something like this:

```
viewControllers = [
        LibraryViewController(),
        WishListViewController(),
        AddBookViewController(),
        RentalsViewController(),
        SettingsViewController()]
```

File: Project/Wishlist/WishListViewController.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//
+
+import Foundation
+import UIKit
+class WishListViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Setting color.
+        self.title = "WishList"
```
Comment (acid-wolox) [inline Project/Wishlist/WishListViewController.swift]: `self` keyword should be used only when strictly needed. In this case, just `title = "WishList"` would do the trick.

File: Project/BaseModel.swift [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+//  Created by Emmanuel Velez Agudelo on 29/10/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+//Generic behavior for all the models
+class BaseModel {
+    
+    
+}
```
Comment (acid-wolox) [inline Project/BaseModel.swift:14]: I don't think you will need this base class, but it won't hurt your app either.

**Review by acid-wolox — CHANGES_REQUESTED** [review]
Nice work, Emma! I left some changes that should/could be made. If you disagree with some of them, feel free to discuss them.

File: Project/ViewController.swift (hunk 2 above)
Comment (emmanuelhcpk) [reply Project/ViewController.swift]: Awesome!

File: Project/BaseModel.swift (hunk 4 above)
Comment (emmanuelhcpk) [reply Project/BaseModel.swift:14]: ok

**Review by acid-wolox — APPROVED** [review]
Looks great!
Congrats on your first PR approval!

**Pull request merged**